laravel-app/app/Http/Requests/UpdateProfileImageRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateProfileImageRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     */
+    public function rules(): array /* @phpstan-ignore-line */
```

Comment:
Why you still have /* @phpstan-ignore-line */ ???

Reply:
I was thinking po of removing it on my next PR since this PR is already crowded and kanina lang po nasettle yung concerns about the @params and @return 🙇

Reply:
removing sir 🙇